### ISSUE TITLE
How to get rid of such log entries?:

### ISSUE THREAD

**Issue by vitalii2011** [Frvetz/FS22_SlipDestroysFields#6]
Error: vehicle type 'FS22_Trex_600.Trex' has unknown specialization 'SlipDestroysFields!

**Comment by Frvetz** [comment]
Hello!

I'll upload an update in the next few days where this problem is fixed.

Thank you for the report! :D

**Comment by Frvetz** [comment]
Hey!

The new version is now available here on github :D

Hope this helps

Kindest regards
Gerrit

**Comment by vitalii2011** [comment]
After updating the mod, new entries appeared (it was this tractor that worked):
```
  dataS/scripts/specialization/SpecializationUtil.lua (23)
  dataS/scripts/vehicles/Vehicle.lua (973) : raiseEvent
  dataS/scripts/i3d/I3DManager.lua (134) : asyncCallbackFunction
2023-04-23 07:55 Error (physics): Wheel shape not found for 'getAxleSpeed', object fastrac4220_axis_component2 wheelShapeIndex 0
2023-04-23 07:55 Error (physics): Wheel shape not found for 'getAxleSpeed', object fastrac4220_axis_component2 wheelShapeIndex 0
2023-04-23 07:55 Error (physics): Wheel shape not found for 'getAxleSpeed', object fastrac4220_main_component1 wheelShapeIndex 0
2023-04-23 07:55 Error (physics): Wheel shape not found for 'getAxleSpeed', object fastrac4220_main_component1 wheelShapeIndex 0
```

**Comment by vitalii2011** [comment]
- Where should the text from "text name="SettingStrengthChanged"", "... SettingDebugChanged", "... SettingDestructionChanged" and "... SettingOtherFieldAccessChanged" appear? 
- Sorry for the screenshot (I'm too lazy to write this :) ): 
![Image 001](https://user-images.githubusercontent.com/41552153/233824854-8722b63f-608b-4ede-aeb0-cbd13cb6842c.png)
- I will offer my version of the Russian translation with minor edits.

**Comment by Frvetz** [comment]
Thank you for the translation!!! And thanks for the info about the other translations, I didn't know that yet...

I'll have a look at the error!

**Comment by vitalii2011** [comment]
Tractor from https://www.farming-simulator.com/mod.php?mod_id=237476

- Just logged in, saved and exited the game: 
[log_0.txt](https://github.com/Frvetz/FS22_SlipDestroysFields/files/11303990/log_0.txt)
- I got into the tractor, but did not start it - the magazine "without errors" (do not pay attention to the WindyLoads mod - I really hope we have already found the reason); 
- Started the tractor - no error;
- I drove a little - there is no error in the log;
- Hooked the roller to the tractor - no error;
- Unfolded, lowered and folded the roller - no error;
- Drove to the field - no error;
- Started to flatten grass in the field, on "inappropriate" tires before reporting overconsolidation of the soil - Error: 
[log_7.txt](https://github.com/Frvetz/FS22_SlipDestroysFields/files/11304022/log_7.txt)

But before and after the update of the mod, this tractor was parked, but the entry nevertheless appeared in the log. I do not know why. But maybe it doesn't matter.

**Comment by Frvetz** [comment]
What tires where you using exactly? That's the only cause I can think of, besides the possibility that another mod is responsible for my script not finding the necessary values

**Comment by vitalii2011** [comment]
In order for the "reconsolidation of the soil" to begin, I set Standard. 

I do not insist, but the log entries appear immediately/only after the message: "Slip Destroys Fields: Ground is being destroyed!".

**Comment by Frvetz** [comment]
Ahh now I have found them in your log. My bad. 
However, I cannot reproduce the error 🤔 Have you tried it without any other mods yet?

**Comment by vitalii2011** [comment]
I'm trying. But I can't find the "hero of the occasion" yet. All "most likely" mods activated. I will activate in batches - I must find it.

**Comment by vitalii2011** [comment]
Repeat on a new map, with the same activated mods, with approximately the same conditions, on the same technique — it did not work. 
With which I congratulate you. :)
The question can be closed.

**Comment by Frvetz** [comment]
So the map was the problem? 
If so, could you tell me the name of the map so I can take a look at it and see if I can fix the error anyway

**Comment by vitalii2011** [comment]
Same vanilla Elmcreek as Errors in the log.

**Comment by Frvetz** [comment]
😄 I‘m a little confused right now.

Did you find the cause of the problem?

**Comment by vitalii2011** [comment]
No, I didn't. 
On Elmcreek, which I have been developing for several weeks, the error records are repeated. (After updating SDF to 1.0.0.3)
On the new Elmcreek, I could not repeat this.
This is the first time I've encountered this and so far there are no ideas. (Not from my practice: fixing errors in one script, "wake up" errors in another mod)
These "Errors" do not interfere with my game.

**Comment by Frvetz** [comment]
Oh okay now I think I understand.

Then I'll close the question now :)

Thank you so much for taking the trouble and opening the question!!!
Have a great week!
Gerrit

**Comment by vitalii2011** [comment]
RU for v1.0.0.5: 
[modDesc.zip](https://github.com/Frvetz/FS22_SlipDestroysFields/files/14901734/modDesc.zip)

**Comment by Frvetz** [comment]
Thank you!
